src/sentry/auth/superuser.py

```diff
@@ -131,6 +131,9 @@ def is_active(self):
         org = getattr(self.request, "organization", None)
         if org and org.id != self.org_id:
             return self._check_expired_on_org_change()
+        # We have a wsgi request with no user.
+        if not hasattr(self.request, "user"):
```

Comment:
Is this the case for RPC calls?

Reply:
Yes, I had to remove authentication (and user loading) for them as the token support we have today requires using an access token from a user. We shouldn't be using a user bound token for these operations though.